charts/r2devops/values.yaml

```
@@ -94,6 +135,22 @@ worker:
   host: ""
   tls: {}
   resources: {}
+  # -- Optional existing service account name to use.
+  serviceAccountName: ""
+  # -- Optional security context for containers.
+  securityContext: {}
+  # -- Optional security context for pods.
+  podSecurityContext: {}
+  automountServiceAccountToken: true
```

Group service account related options

```suggestion
  # -- Optional existing service account name to use.
  serviceAccountName: ""
  automountServiceAccountToken: true
  # -- Optional security context for containers.
  securityContext: {}
  # -- Optional security context for pods.
  podSecurityContext: {}
```